docs/kbs_protocol.md

```diff
@@ -0,0 +1,276 @@
+# KBS Protocol
+
+As a trusted service, KBS's client (Key Broker Client, KBC for short) runs in a HW-TEE. 
+When the KBC sends a request to KBS, HW-TEE attestation is required. Universal-KBS performs a 
+simple, universal and extensible "Request-Challenge-Attestation-Response" (we call it **RCAR** 
+for short) protocol to complete the attestation, and encrypt the response payload content 
+returned by KBS to KBC with the key which protected by HW-TEE.
```

Comment:
I would replace this with: 

The Key Broker protocol defines communication between a Key Broker Client in a confidential guest and a trusted Key Broker Service. The protocol uses the simple, universal, and extensible "Request-Challenge-Attestation-Response" (RCAR) method to facilitate guest attestation and secret injection.